### DISCUSSION TITLE
any way to tell what fields a method accesses.

### DISCUSSION THREAD

**Discussion by aarnithesuperproguyprosuperguypropro** [vfsfitvnm/frida-il2cpp-bridge#467] [Q&A]
for example
private uint value;
private bool something = false;
void checkForSomething(){
    something = (value >= 130);
}
Is there a way to know what fields the method accesses?

**Comment by RevealedSoulEven** [comment]
Please tell me too if you get

**Comment by thinhbuzz** [comment]
Use ida or ghidra or similar app to open il2cpp

**Reply by RevealedSoulEven** [reply]
But what if we need to know at runtime?

**Reply by Flechaa** [reply]
You can ``console.log`` the class and it will print out the methods, fields, etc.

**Reply by RevealedSoulEven** [reply]
Aahhhhhh! We're talking about method, not class.

**Reply by Flechaa** [reply]
So you're trying to trace the class? That's what I understood.

**Reply by RevealedSoulEven** [reply]
Not tracing class, like suppose a class have many methods, you can say we want to know which fields are being called by a method, like we can see which methods are being called by a method, but what are the fields it's using.

I know it can be done by IDA and Ghidra, but it would be lot compatible if could be viewed runtime.

**Reply by aarnithesuperproguyprosuperguypropro** [reply]
I'm trying to trace a method. I'm not sure if that's possible though.

**Reply by RevealedSoulEven** [reply]
If you get to know please take out some time to explain me too
